hack/deploy-cluster-capz.sh

```diff
@@ -40,6 +40,7 @@ export AZURE_LOCATION="${AZURE_LOCATION:-westus2}"
 export AZURE_CLOUD_CONTROLLER_MANAGER_IMG="${AZURE_CLOUD_CONTROLLER_MANAGER_IMG:-mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.23.1}"
 export AZURE_CLOUD_NODE_MANAGER_IMG="${AZURE_CLOUD_NODE_MANAGER_IMG:-mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.1}"
 export KUBERNETES_VERSION="${KUBERNETES_VERSION:-v1.23.0}"
+export IN_TREE="${IN_TREE:-false}"
```

Comment:
Can we make it `USE_IN_TREE_CLOUD_PROVIDER`?

Reply:
Updated.